fiftyone/core/dataset.py

```diff
@@ -3336,41 +3356,22 @@ def add_collection(
         )
         return self.skip(num_samples).values("id")
 
-    def _add_samples_batch(
-        self, samples, expand_schema, dynamic, validate, batcher=None
-    ):
-        samples = [s.copy() if s._in_db else s for s in samples]
-
-        if self.media_type is None and samples:
-            self.media_type = _get_media_type(samples[0])
-
-        if expand_schema:
-            self._expand_schema(samples, dynamic)
-
-        if validate:
-            self._validate_samples(samples)
-
-        now = datetime.utcnow()
-        dicts = [
-            self._make_dict(sample, created_at=now, last_modified_at=now)
-            for sample in samples
-        ]
-
+    def _add_samples_batch(self, samples, batcher=None):
+        dicts = [sample[1] for sample in samples]
```

Comment:
I was very confused until I realized samples is tuple(sample, transformed_dict).... this is where type hints would be 🤌

Reply:
Yeah can we add a comment at least?